index.html

```diff
@@ -3,15 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="/favicon.png" />
-    <link
```

Comment:
it could break fonts in snackbar 
![image](https://github.com/Visual-Regression-Tracker/frontend/assets/5182956/89ad2f1d-d79e-44d2-a038-d084ddbccd48)

Reply:
Fixed by setting font family in global css